test-data/run.test

```diff
@@ -611,3 +611,16 @@ sum = 0.0
 for i in range(10):
     sum += assign_and_return_float_sum()
 assert sum == 50.0
+
+[case testBigIntLiteral]
+def big_int() -> None:
+    a_62_bit = 4611686018427387902
+    max_62_bit = 4611686018427387903
+    b_63_bit = 4611686018427387904
+    c_63_bit = 9223372036854775806
+    max_63_bit = 9223372036854775807
+    d_64_bit = 9223372036854775808
+    max_32_bit = 2147483647
+[file driver.py]
+from native import big_int
+big_int()
```

Comment:
Have the test either print or return some values and check that they are correct.

Reply:
Done.